### PR TITLE
Links to connect

### PR DESCRIPTION
**Pull request by vijaytembugade**
Added readme.md files
Added links to connect each other
Minor changes

README : https://github.com/vijaytembugade/Ecommerce-Screens/blob/Links_to_connect/readme.md

### CODE REVIEW

File: Screens/Filter/filter.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .plp-container {
-  margin: 5rem 0rem 0rem 0rem;
+  margin: 5rem 0rem 0rem ;
   display: flex;
 }
```
Comment (kshitijthatte) [inline Screens/Filter/filter.css:4]: You can use `margin-top` here since other values are 0.

**Review by kshitijthatte — APPROVED** [review]
Looks great!

**Review by TusharJade — APPROVED** [review]
Looks good to me, good to go👍

**Review by msdeshmukh009 — APPROVED** [review]

**Review by horsemaker — APPROVED** [review]
The README looks apt. Has everything that should be needed! 👌

**Pull request merged**